exercises/numbers/celsiusToFahrenheit/celsiusToFahrenheit.js

```diff
@@ -5,13 +5,20 @@
  * @returns {boolean} The temperature in Fahrenheit
  */
 function celsiusToFahrenheit(temp) {
-  // This is your job. :).
+{
+var cTemp = celsius;
```

Comment:
Seems like you're copying your code from https://www.w3resource.com/javascript-exercises/javascript-basic-exercise-11.php

When you copy code, make sure you try to understand it. If you don't understand parts of it, direct our attention to which parts you're trying to make sense of and the external resource you used.

The code as written won't work. Use `node` to run it by navigating to the `numbers` directory and running:

```console
node celsiusToFahrenheit/celsiusToFahrenheit.js
```

The code as written doesn't work and even if the variables were named correctly, it wouldn't do what the exercise was asking.